src/hyperion/log.py

```diff
@@ -52,6 +55,16 @@ def do_default_logging_setup(dev_mode=False):
     integrate_bluesky_and_ophyd_logging(dodal_logger, handlers)
     handlers["graylog_handler"].addFilter(dc_group_id_filter)
 
+    global __logger_handlers
+    __logger_handlers = handlers
+
+
+def get_memory_handler() -> CircularMemoryHandler:
```

Comment:
Nit: I think `get_debug_handler` would be slightly clearer as it's logging the debug not the memory